packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/connectors.html

```diff
@@ -0,0 +1,166 @@
+            data-property="name"
+            data-sortable="asc"
+            data-type="string"
+            data-searchable
+            data-filterable
+        ></oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_connectors_connector' | translate"
+            data-property="connectorInformation.name"
+            data-sortable="asc"
+            data-type="string"
+            data-searchable
+            data-filterable
+        >
+            <div class="d-flex align-items-center">
+                <span
+                    class="round-icon oui-icon mr-2 oui-background-p-400 oui-color-p-050"
+                    ng-class="{
+                    'oui-icon-arrow-down': $row.connectorInformation.type === $ctrl.CONNECTOR_TYPES.SINK,
+                    'oui-icon-arrow-up': $row.connectorInformation.type === $ctrl.CONNECTOR_TYPES.SOURCE
+                    }"
+                >
+                </span>
+                <div>
+                    <h6
+                        ng-bind="$row.connectorInformation.name"
+                        class="mb-0"
+                    ></h6>
+                    <span class="oui-paragraph">
+                        <span
+                            >Version:
+                            {{$row.connectorInformation.version}}</span
+                        >
+                        <span>
+                            | Author: {{$row.connectorInformation.author}}</span
+                        >
+                        <span
+                            ng-if="$row.connectorInformation.documentationUrl"
+                        >
+                            |
+                            <a
+                                ng-href="{{$row.connectorInformation.documentationUrl}}"
+                                target="_blank"
+                                rel="noopener"
+                                >docs</a
+                            >
+                        </span>
+                    </span>
+                </div>
+            </div>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_connectors_status' | translate"
+            data-property="status"
+            data-sortable="asc"
+            data-type="string"
+            data-searchable
+            data-filterable
+        >
+            <span
+                class="oui-badge"
+                data-ng-class="{
+                    'oui-badge_success': !$row.isPaused(),
+                    'oui-badge_warning':  $row.isPaused() ||  $row.isCreating()
+                }"
+            >
+                <span>{{$row.status}}</span>
+            </span>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_connectors_tasks' | translate"
+            data-property="tasks"
+            data-sortable="asc"
+            data-type="string"
+        >
+            <div
```

Comment:
**suggestion**:

1. Add a `getSucceededTasks` method to the class
2. The template can be simplified to:
```
<div class="d-flex align-items-center" data-ng-class="$row.getSucceededTasks().length === $row.tasks.length" ? 'oui-color-as-500' : 'oui-color-ae-500'">
          {{$row.getSucceededTasks().length}}/{{$row.tasks.length}}
</div>
```

Reply:
I can't simplify the template here because of the button to go to tasks which only appears when there is at least one failed task

Reply:
How about adding an `data-ng-if` on `button`